packages/components/src/color-palette/index.js

```diff
@@ -162,6 +162,10 @@ export const extractColorNameFromCurrentValue = (
 	return __( 'Custom' );
 };
 
+export const showTransparentBackground = ( currentValue ) => {
+	return colord( currentValue ).alpha() === 0;
+};
```

Comment:
Can we add handling (and a test case) for an `undefined` color value? It will be undefined when the "Clear" button is clicked.

Reply:
> Can we add handling (and a test case) for an `undefined` color value? It will be undefined when the "Clear" button is clicked.

@mirka Yep, did with d482da9668142d7b8fefb39ec2f59487c9d11b10